### DISCUSSION TITLE
Is it time to update USA banner's content? (Security question)

### DISCUSSION THREAD

**Discussion by zarahzachz** [uswds/uswds#5411] [Ideas]
I'm working on a USABanner component for CMS and was struck with a content question... should [the banner](https://designsystem.digital.gov/components/banner/) content be updated to remove references to the lock icon? 

[Chrome is removing this icon in September](https://www.engadget.com/google-is-replacing-chromes-lock-icon-because-most-people-dont-know-what-it-means-194503194.html) due to the following:

> According to Google's research, only 11 percent of users realize that it refers to HTTPS encryption. Many others think it means the site is trustworthy — a problem when even phishing sites use the technology.

This is a question I'm fielding on the CMS side, but didn't know if y'all would benefit from this discussion as well.

**Comment by zarahzachz** [comment]
More information: [Krebs on Security reported in 2018 that half of phishing sites use a padlock in the url](https://krebsonsecurity.com/2018/11/half-of-all-phishing-sites-now-have-the-padlock/)

So even if Chrome's decision to remove the lock isn't a big deal, it seems like the banner telling users to look for this symbol as a sign the site's trustworthy deserves some scrutiny.

**Comment by thisisdano** [comment]
I think this case is a little bit tricky, since the message we want to convey is that folks should be looking for a combination of a `.gov` or `.mil` domain _and_ `https`. For now, I think it's still accurate and relevant that a padlock is an indicator of `https`, though — as you say — it's appearing less and less. (Tends to be moving toward appearing as an active warning when https is _not_ present.)

Content-wise, the line `Secure .gov websites use HTTPS` is the key message — not to trust a padlock or https in isolation. 

I think, too, that we're moving toward a future where there's more of a direct connection between `.gov`/`.mil` and `https` and this kind of message is less and less necessary. But for now, the banner message is accurate and necessary.
